docs/ldml/tr35.md

```diff
@@ -119,6 +119,7 @@ The LDML specification is divided into the following parts:
     * [Lateral Inheritance](#Lateral_Inheritance)
       * Table: [Count Fallback: normal](#Count_Fallback_normal)
       * Table: [Count Fallback: currency](#Count_Fallback_currency)
+    * [Inheritance Marker](#inheritance-marker)
```

Comment:
These changes might be because Steven hasn't rerun the TOC generator?? @srl295 can you check on that?

Reply:
Yep I was working on running it today as part of fixing the anchor issue